.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches: [master]
```

Comment:
Perhaps we can make these run on all branches and all PRs? That way, we don't assume people's name of the default branch.

Reply:
Agreed - I made it so it'll run whenever someone pushes :)